src/caja-application.c

```diff
@@ -1830,6 +1837,55 @@ do_perform_self_checks (gint *exit_status)
     *exit_status = EXIT_SUCCESS;
 }
 
+static void
+select_items_ready_cb (GObject *source,
+               GAsyncResult *res,
+               gpointer user_data)
```

Comment:
Thanks, but in new or changed code the indents are wrong.
Sigh, i didn't mentioned that explicitly before.